components/director/cmd/ns-adapter/README.md

```diff
@@ -0,0 +1,313 @@
+# NS-Adapter
+
+## Overview
+
+NS-Adapter listens for incoming **reports** on on-premice systems from external Notification Service. Based on the data from the report systems may be created, updated or deleted.
+
+## Details
+
+This section describes the API schema that a Notification Service must implement to integrate with the NS-Adapter.
+
+### Authorization
+
+Valid JWT token with "tenant.consumerTenant" and "tenant.externalTenant" claims. ConsumerTenant and externalTenant should contain the internal ID and the external ID from existing business_tenant_mappings record in Compass database.
+
+### Reports
+
+There are two types of reports:
+- Delta:
+  - systems that are __present__ in the report but are __unknown__ to Compass will be created
```

Comment:
Since you mention below the `SCC` label, maybe it would be good to note, that the systems will be created and labeled with `SCC` which consists of subaccount_id, location_id, host